tests/fast-integration/skr-test/skr-test.js

```diff
@@ -20,18 +20,29 @@ const {
   ensureCommerceMockWithCompassTestFixture,
   checkAppGatewayResponse,
   sendEventAndCheckResponse,
+  deleteMockTestFixture,
 } = require("../test/fixtures/commerce-mock");
 const {
   debug,
   genRandom,
   initializeK8sClient,
 } = require("../utils");
 
+const { 
+  Creds,
+  AuditLogClient,
+  waitForAuditLogs,
+  checkAuditLogs
+} = require("../audit-log");
+
 describe("SKR test", function() {
   const keb = new KEBClient(KEBConfig.fromEnv());
   const gardener = new GardenerClient(GardenerConfig.fromEnv());
   const director = new DirectorClient(DirectorConfig.fromEnv());
 
+  const cred = new AuditLogClient(Creds.fromEnv())
```

Comment:
This represents the AditLogClient not creds, and please rename the Creds class to something more meaningful in context of audit logs (eg. AuditLogCredentials). Now it might be obvious but after some time might be misleading.

```suggestion
  const auditlogs = new AuditLogClient(Creds.fromEnv())
```

Reply:
fixed